sydent/sms/openmarket.py

```diff
@@ -161,11 +161,3 @@ async def sendTextSMS(
             request_id,
             resp.code,
         )
-
```

Comment:
Should the above `logger.error` lines be accompanied with a `return`?

Reply:
Yes. What the dickens is happening here?

Reply:
Tempted as I am to rewrite the whole thing, I'll be a good boy: stick in the return and call it a day.